sdk/python/kfp/v2/components/experimental/base_component_test.py

```diff
@@ -80,7 +80,7 @@ def test_instantiate_component_omitting_arguments_with_default(
         arguments={
             'input1': 'hello',
             'input2': 100,
-            'input3': 3.14,
+            'input3': '3.14',
```

Comment:
why is this a string? (yet the integer default is not a string)

Reply:
As discussed offline, it's a default field which gets parsed to string originally, will think about future improvement.